bears/general/FilenameBear.py

```diff
@@ -19,14 +19,22 @@ class FilenameBear(LocalBear):
                           'snake': to_snakecase,
                           'space': to_spacecase}
 
+    _language_naming_convention = {
+                                   '.java': 'pascal',
+                                   '.js': 'kebab',
+                                   '.py': 'snake',
+                                  }
+
     def run(self, filename, file,
-            file_naming_convention: str='snake',
+            file_naming_convention: str=None,
             ignore_uppercase_filenames: bool=True):
         """
         Checks whether the filename follows a certain naming-convention.
 
         :param file_naming_convention:
             The naming-convention. Supported values are:
+            - ``auto`` to guess the correct convention. Defaults to
+            ``snake`` if the correct convention cannot be guessed.
```

Comment:
Did not align because it caused indentation problems https://circleci.com/gh/coala/coala-bears/5890#tests/containers/1